src/bp/core/routers/bots/index.ts

```diff
@@ -119,10 +119,11 @@ export class BotsRouter extends CustomRouter {
     }
   }
 
-  getNewRouter(path: string, options?: RouterOptions) {
+  getNewRouter(path: string, owner: string, options?: RouterOptions) {
     const router = Router({ mergeParams: true })
     if (_.get(options, 'checkAuthentication', true)) {
       router.use(this.checkTokenHeader)
+      router.use(this.needPermissions('read', `module.${owner}`))
```

Comment:
I'd make it `write` since we don't guard against write for now, so that it's obvious . (because write means you also have read)

Reply:
Good point, will do